src/taco-cli/cli/utils/platformPluginCommandBase.ts

```diff
@@ -146,6 +153,9 @@ export class PlatformPluginCommandBase extends commands.TacoCommandBase {
             return cordovaWrapper.invokePlatformPluginCommand(self.name, self.cordovaCommandParams, data);
         })
             .then(function (): Q.Promise<any> {
+            return self.postInvokeStep(data);
```

Comment:
Currently all "taco platform X" and "taco plugin X" commands go through this class doing the same steps, eventually deferring to corodvaWraper.invokePlatformPluginCommand just above. If we want to refactor these classes to use the subcommand style that the other commands use, it'll be a bigger task.